markdown/org/docs/patterns/waralee/cutting/en.md

```diff
@@ -1,18 +1,37 @@
-Waralee consists of two identical pieces of fabric.
+<Note>
+
+Waralee consists of two mirrored pieces of fabric.
 They are basically big rectangles with pieces missing.
-As such, it is easier to print out the piece you need to cut out and measure the rest,
+As such, it is easier to print out the part you need to cut out and measure the rest,
 than it is to print out everything and cut the pattern the traditional way.
 
-The pattern is available both ways.
-You can print out the whole thing, tape all the pages together, and use that to cut out the fabric.
-It is also possible to print out only the part that you need to cut out.
-This part has a negative seam allowance, since you'll be using it opposite of what you normally use the pattern pieces for.
+That said, the pattern is available both ways. So you can print out the whole thing, tape all the pages together, and use that to cut out the fabric, if you prefer. But we recommend only printing the part that you need to cut out.
+
+You will need to print out the pockets and pocket facings in the traditional way.
+
+</Note>
+
+### Materials
+ - **Main fabric**
+   - Cut the **2 pants** parts.
+   - (Optional) Cut **2 Pocket** parts on the fold.
+   - Cut **4 backPocket** parts
+   - Cut **4 frontFacing** parts
+   - Cut **4 backFacing** parts
+
+<Note>
+
+You can cut the Pocket and backPocket out of a lining fabric if you prefer but this with only be needed if you deviate from the recommended main fabrics.
```

Comment:
Small typo and a suggestion for making the number of main fabrics (1 I think) clearer:
```suggestion
You can cut the Pocket and backPocket out of a lining fabric if you prefer but this will only be needed if you deviate from the main fabric recommendations.
```

Reply:
I think just the will bit